docs/en/query_language/create.md

```diff
@@ -1,13 +1,33 @@
-## CREATE DATABASE
+## CREATE DATABASE {#query_language-create-database}
 
-Creating db_name databases
+Creates database.
 
 ``` sql
-CREATE DATABASE [IF NOT EXISTS] db_name [ON CLUSTER cluster]
+CREATE DATABASE [IF NOT EXISTS] db_name [ON CLUSTER cluster] [ENGINE]
 ```
 
-`A database` is just a directory for tables.
-If `IF NOT EXISTS` is included, the query won't return an error if the database already exists.
+### Clauses
+
+- `IF NOT EXISTS`
+
+    If the `db_name` database already exists then:
+
+    - If clause is specified, ClickHouse doesn't creates a new database and doesn't throw an exception.
+    - If clause is not specified, then ClickHouse doesn't creates a new database and throw and exception.
+
+- `ON CLUSTER`
+
+    ClickHouse creates the `db_name` database on all the servers of a cluster.
+
+- `ENGINE`
+
+    - [ClickHouse](../database_engines/clickhouse.md)
+
+        This is the default database engine. Should never be specified in the query.
```

Comment:
"Should never be specified in the query." — why? isn't it just useless, but not harmful?